cmd/bootkube/start.go

```diff
@@ -21,19 +21,22 @@ var (
 	startOpts struct {
 		assetDir        string
 		podManifestPath string
+		strict          bool
 	}
 )
 
 func init() {
 	cmdRoot.AddCommand(cmdStart)
 	cmdStart.Flags().StringVar(&startOpts.assetDir, "asset-dir", "", "Path to the cluster asset directory. Expected layout genereted by the `bootkube render` command.")
 	cmdStart.Flags().StringVar(&startOpts.podManifestPath, "pod-manifest-path", "/etc/kubernetes/manifests", "The location where the kubelet is configured to look for static pod manifests.")
+	cmdStart.Flags().BoolVar(&startOpts.strict, "strict", false, "Strict mode.")
```

Comment:
nit: More of a description than just "Strict mode". Maybe

> When run in strict mode bootkube fails if it can't create any of the manifests in the asset directory.